common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -61,6 +62,7 @@ public TransportConf(String module, ConfigProvider conf) {
     SPARK_NETWORK_IO_MAXRETRIES_KEY = getConfKey("io.maxRetries");
     SPARK_NETWORK_IO_RETRYWAIT_KEY = getConfKey("io.retryWait");
     SPARK_NETWORK_IO_LAZYFD_KEY = getConfKey("io.lazyFD");
+    SPARK_NETWORK_VERBOSE_METRICS = getConfKey("io.enableVerboseMetrics");
```

Comment:
Add a configuration to control whether to expose detailed metrics. Normally detailed per arena metrics will not be tracked, user could enable this through configuration, so that user has the ability to know the details of Netty memory allocation behavior, which could potentially guid to fine-grained tune.